src/main/java/teammates/common/datatransfer/FeedbackSessionLogEntry.java

```diff
@@ -1,26 +1,57 @@
 package teammates.common.datatransfer;
 
+import java.util.UUID;
+
 /**
  * Represents a log entry of a feedback session.
  */
 public class FeedbackSessionLogEntry implements Comparable<FeedbackSessionLogEntry> {
+    private final String courseId;
+    private final UUID studentId;
     private final String studentEmail;
+    private final UUID feedbackSessionId;
     private final String feedbackSessionName;
```

Comment:
Is it possible to drop the `studentEmail` and `feedbackSessionName` now that we have the entities itself?